nservicebus/upgrades/5to6/distributor.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Deprecated Distributor in NServiceBus Version 6
```

Comment:
@timbussmann - I am good with `Migrating the Distributor to use Sender Side Distribution`